scripts/time-drift.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# zenoss-inspector-tags verify
+#
+
+for i in $(serviced host list| awk '{print $4}' | grep -v Name | grep -v $HOSTNAME); do echo ntpdate \-q $i; ntpdate -q $i; done
```

Comment:
Thank you, 
I have updated the code to use the IP address for connectivity. 
I updated the code to filter out the host-name and that does look better.
I updated the code to use multiply lines and it is easier to read.
I updated the code to have an exit 0 of zero. 
I like your changes thank you.